backend/src/main/java/gov/cdc/usds/simplereport/api/InternalIdResolver.java

```diff
@@ -1,12 +1,9 @@
 package gov.cdc.usds.simplereport.api;
 
 import gov.cdc.usds.simplereport.db.model.DatabaseEntity;
-import graphql.kickstart.tools.GraphQLResolver;
 import java.util.UUID;
 
 /** Resolver mix-in to alias "id" to "internalId" */
-public interface InternalIdResolver<T extends DatabaseEntity> extends GraphQLResolver<T> {
-  default UUID getId(T entity) {
-    return entity.getInternalId();
-  }
+public interface InternalIdResolver<T extends DatabaseEntity> {
+  UUID getId(T entity);
```

Comment:
see [this comment](https://github.com/CDCgov/prime-simplereport/pull/4243/files#r966571178) for more details on this change